.github/workflows/packager_deb.yml

```diff
@@ -0,0 +1,126 @@
+name: packager_deb
+
+on:
+  push:
+    branches:
+      - 'main'
+    paths:
+      - '**'
+    tags:
+      - 'v*.*.*'
+      - 'v*.*.*-*'
+
+jobs:
+  build:
+    permissions:
+      id-token: write
+      contents: write
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up Go
+        uses: actions/setup-go@master
+        with:
+          go-version: 1.20.x
```

Comment:
Should the version here be the same as the [version](https://github.com/maticnetwork/bor/blob/master/go.mod#L3) in go.mod?

Reply:
Updated the version